index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Maria S. Resume</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<base target="blank">
+</head>
+<body>
+
+<header>
+		<h1>Maria S.</h1>
+		<nav>
+			<h2><a href="index.html">Resume</a> |
+			<a href="interests.html">Interests</a></h2>
+		</nav>
+	</header>
+
+<br>
+
+<hr>
+
+<h2>Education</h2>
+
+<p><b>[2004-2008] University of Pennsylvania & The Wharton School </b><br>
```

Comment:
Use of presentational tags such as `b` should be avoided when possible in favor of a CSS class, which is current best practice.  🔨